modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -437,23 +437,23 @@ private static boolean anyOperationParam(final List<CodegenOperation> operations
         final Set<String> dependencies = new HashSet<>();
 
         for (CodegenOperation op : ops) {
-            if (ElmVersion.ELM_018.equals(elmVersion)) {
+            if (ElmVersion.ELM_018.equals(elmVersion)) { // elm 0.18
                 String path = op.path;
                 for (CodegenParameter param : op.pathParams) {
                     final String var = paramToString("params", param, false, null);
                     path = path.replace("{" + param.paramName + "}", "\" ++ " + var + " ++ \"");
```

Comment:
Did this change? I guess this should become `baseName` as well then.

Reply:
Let me update that too.

Reply:
Update via e417b421e4